packages/components/src/tools-panel/tools-panel/README.md

```diff
@@ -73,19 +69,22 @@ export function DimensionPanel( props ) {
 
 ### `label`: `string`
 
-The label for the panel's dropdown menu.
+Text to be displayed within the panel's header and as the aria label for the
+panel's dropdown menu.
 
 - Required: Yes
 
-### `resetAll`: `function`
+### `panelId`: `function`
```

Comment:
> Also, do you think it'd be valuable to add a Storybook example showing how to use the panelId props, especially in the context of injection of panel items via a SlotFill?

Continuing on the comment from @ciampo, it feels like this prop needs a better description that is disconnected from the SlotFill pattern. `@wordpress/components` is a set of general purpose components so it feels like this part is tightly couple with the higher-level abstraction that doesn't fit here.

Reply:
I've had another go at [describing this prop](https://github.com/WordPress/gutenberg/pull/34632/commits/ee6b6821b9a5960f56ec3c8e38f513c31600c58a) free of mentioning the SlotFill pattern. Let me know if that misses the mark and needs some further rewording.

Reply:
Thank you. I like more general phrasing more 👍🏻